CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Jinja Filters Changelog
 =======================
 
+- **feature** add ``merge_date_url`` filter
+- **feature** add ``datetime_from_period`` filter
+- **support** rework supporting ``tasks.py`` (for use with ``invoke``)
+- **support** add test suite
+- **support** support Pelican from version 3.0 on
+
```

Comment:
The core design philosophy behind AutoPub includes never needing to manually add entries to the change-log. These entries should instead be in the form of a `RELEASE.md` file:
```markdown
Release type: patch

- **feature** add ``merge_date_url`` filter
- **feature** add ``datetime_from_period`` filter
- **support** rework supporting ``tasks.py`` (for use with ``invoke``)
- **support** add test suite
- **support** support Pelican from version 3.0 on
```

Reply:
@justinmayer Done! I've moved this to a `RELEASE.md` file.